remote.go

```diff
@@ -126,17 +155,38 @@ func toPartialError(temporary bool, rcpts []string, err error) *PartialError {
 	return &perr
 }
 
+var ErrNoMXMatchedBySTS = errors.New("remote: no MX record matched MTA-STS policy")
+
 func (rd *RemoteDelivery) deliverForServer(ctx *module.DeliveryContext, domain string, rcpts []string, body io.Reader) *PartialError {
 	addrs, err := lookupTargetServers(domain)
 	if err != nil {
 		return toPartialError(false, rcpts, err)
 	}
 
+	stsPolicy, err := rd.mtastsCache.Get(domain)
+	if err != nil && err != mtasts.ErrNoPolicy {
+		rd.Log.Printf("failed to fetch MTA-STS policy for %s: %v", domain, err)
+		// Problems with policy should be threated as temporary ones.
```

Comment:
Nit: typo in treated

Reply:
I should definitely get a spell checker into my code editor.